packages/react-server-website/deployment/deploy.yml

```diff
@@ -11,10 +11,18 @@
     - name: Copy NGINX configuration
       copy: src=../nginx.conf dest=/home/ubuntu/nginx.conf mode=0644
 
-    - name: Pull containers
-      shell: docker-compose pull && docker-compose build
+    - name: Pull images
+      command: docker-compose pull
```

Comment:
Yes, the `build` step was an attempt to address #511 before I had reproduced it or understood what was going on. In fact, doing a `build` there has no effect because we are deploying pre-built images.